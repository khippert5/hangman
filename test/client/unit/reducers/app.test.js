import appReducer, { defaultState } from '../../../../src/client/reducers/app';

describe('Test ui reducer', () => {
  // Ensuer the solution is always the same on time of start
  const solution = {
    word: 'test',
    hint: 'Test answer',
  }

  it('Match snapshot to ensure changes are verified', async (done) => {
    expect(defaultState).toMatchSnapshot()
    done();
  });
  it('INIT_APPLICATION', async (done) => {
    const result = appReducer(defaultState, { type: 'INIT_APPLICATION'});
    expect(result.init.date).not.toBe(null);
    done();
  })
})
