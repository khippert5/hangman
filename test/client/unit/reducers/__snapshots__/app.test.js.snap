// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ui reducer Match snapshot to ensure changes are verified 1`] = `
Object {
  "init": Object {
    "date": null,
    "time": null,
  },
}
`;
